solidity/test/Coordinator_test.js

```diff
@@ -13,30 +14,30 @@ import {
   getLatestEvent,
   initiateServiceAgreement,
   initiateServiceAgreementCall,
-  newAddress,
-  newHash,
+  linkContract,
+  keccack,
   newServiceAgreement,
   oracleNode,
   pad0xHexTo256Bit,
   padNumTo256Bit,
   personalSign,
   recoverPersonalSignature,
-  requestDataBytes,
-  requestDataFrom,
   runRequestId,
   sixMonthsFromNow,
   stranger,
   strip0x,
   toHex,
-  toWei
+  toUtf8,
+  toWei,
 } from './support/helpers'
```

Comment:
Likewise here. 30+ lines of imports makes me think `import * as h from './support/helpers'`.

Reply:
d4687ded